### PR TITLE
Refresh network-fritzbox-upnp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-fritzbox-upnp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md [hunk 1]
```diff
@@ -5,32 +5,55 @@ title: Fritz!Box UPnP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md [hunk 2]
```diff
@@ -5,32 +5,55 @@ title: Fritz!Box UPnP
...
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Fritzbox-UPNP-custom" label="Net-Fritzbox-UPNP-custom">
+
+| Alias   | Modèle de service                | Description                                   |
+|:--------|:---------------------------------|:----------------------------------------------|
+| System  | Net-Fritzbox-System-UPNP-custom  | Contrôle le système                           |
+| Traffic | Net-Fritzbox-Traffic-UPNP-custom | Contrôle le traffic réseau de l'interface WAN |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:31]:
```suggestion
| Traffic | Net-Fritzbox-Traffic-UPNP-custom | Contrôle le trafic réseau de l'interface WAN |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md [hunk 3]
```diff
@@ -39,87 +62,290 @@ Le Pack Fritz!Box collecte les données pour:
 
 Afin de contrôler votre équipement Fritz!Box, l'UPnP doit être configuré.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:63]: ?

File: pp/integrations/plugin-packs/procedures/network-fritzbox-upnp.md [hunk 4]
```diff
@@ -5,32 +5,54 @@ title: Fritz!Box UPnP
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Fritzbox-UPNP-custom" label="Net-Fritzbox-UPNP-custom">
+
+| Service Alias | Service Template                 | Service Description         |
+|:--------------|:---------------------------------|:----------------------------|
+| System        | Net-Fritzbox-System-UPNP-custom  | Checks system               |
+| Traffic       | Net-Fritzbox-Traffic-UPNP-custom | Check wan traffic interface |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:30]:
```suggestion
| Traffic       | Net-Fritzbox-Traffic-UPNP-custom | Check wan interface traffic |
```

File: pp/integrations/plugin-packs/procedures/network-fritzbox-upnp.md [hunk 5]
```diff
@@ -39,86 +61,290 @@ The Pack Fritz!Box collects metrics for:
 
 To control your Fritz!Box, the UPnP must be configured.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:62]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md (hunk 3 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-fritzbox-upnp.md:63]:
```suggestion
Afin de contrôler votre équipement Fritz!Box, l'UPnP doit être configuré. Référez vous à la [documentation officielle](https://en.avm.de/service/manuals/fritzbox/).
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**